app/uk/gov/hmrc/gform/sharedmodel/formtemplate/ComponentType.scala

```diff
@@ -564,7 +564,7 @@ object Range {
         getTimeSlots(t.startTime.time, t.endTime.time, time.intervalMins.intervalMins, List(t.startTime.time))
       )
       .distinct
-      .map(_.format(twelveHoursFormat))
+      .map(_.format(twelveHoursFormat).replace("AM", "am").replace("PM", "pm"))
```

Comment:
This was added because Alex H brought up that it shows as capitalised for him. Might be operating system dependent, as on my Mac system, it's already lowercase.